azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/handlers/DeploymentSlotHandlerImpl.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.maven.webapp.handlers;
+
+import com.microsoft.azure.management.appservice.DeploymentSlot;
+import com.microsoft.azure.management.appservice.WebApp;
+import com.microsoft.azure.maven.auth.AzureAuthFailureException;
+import com.microsoft.azure.maven.webapp.AbstractWebAppMojo;
+import com.microsoft.azure.maven.webapp.configuration.DeploymentSlotSetting;
+import org.apache.maven.plugin.MojoExecutionException;
+
+public class DeploymentSlotHandlerImpl implements DeploymentSlotHandler {
```

Comment:
will there be other handlers other than this?

Reply:
currently no, and this makes it more like a Util class.

Reply:
Rethink about it, make DeploymentSlotUtils to handle deployment get/check/create functionalities.